pkg/apis/testharness/v1beta1/test_types.go

```diff
@@ -25,6 +25,9 @@ type TestSuite struct {
 	// ControlPlaneArgs defaults to APIServerDefaultArgs from controller-runtime pkg/internal/testing/integration/internal/apiserver.go
 	// this allows for control over the args, however these are not serialized from a TestSuite.yaml
 	ControlPlaneArgs []string
+	// AttachControlPlaneOutput if true, attaches control plane logs (api-server, etcd) into stdout. This is useful for debugging.
+	// defaults to false
+	AttachControlPlaneOutput bool
```

Comment:
```suggestion
	AttachControlPlaneOutput bool `json:"attachControlPlaneOutput"`
```

This should have a json mapping, right? I'm not sure why the controlPlaneArgs above don't have it...

Reply:
thanks! I've fixed it and added it to the above one as well